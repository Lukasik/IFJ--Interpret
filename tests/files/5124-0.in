hovno
hovno
ahoj
1.0
9.0
100.0
10.0
a
a
a
