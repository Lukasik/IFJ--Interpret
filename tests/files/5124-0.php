<?php


function neni10()
{
	$q=get_string();
	$convert=doubleval($q);
	return ($convert!==10.0);
}


$res=neni10();
while ($res)
{
	$q=put_string($res,"\n");
	$res=neni10();
}
$q=put_string("konec vole");