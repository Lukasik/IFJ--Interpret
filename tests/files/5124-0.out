1
1
1
1
1
1
konec vole